x-pack/platform/plugins/shared/cases/server/plugin.ts

```diff
@@ -211,12 +209,20 @@ export class CasePlugin
       scheduleCasesTelemetryTask(plugins.taskManager, this.logger);
 
       if (this.caseConfig.analytics.index?.enabled) {
-        scheduleCasesAnalyticsSyncTasks({ taskManager: plugins.taskManager, logger: this.logger });
+        const internalSavedObjectsRepository = core.savedObjects.createInternalRepository([
```

Comment:
nit (for a potential cleanup): we can move this logic into the `createCasesAnalyticsIndexes` since it's only used there